src/connection_string.ts

```diff
@@ -166,14 +165,14 @@ export function resolveSRVRecord(options: MongoOptions, callback: Callback<HostA
 /**
  * Checks if TLS options are valid
  *
- * @param options - The options used for options parsing
- * @throws MongoParseError if TLS options are invalid
+ * @param allOptions - all options provided by user
+ * @throws MongoAPIError if TLS options are invalid
  */
-export function checkTLSOptions(options: AnyOptions): void {
```

Comment:
Remove `export` because AFAICT `checkTLSOptions` is not used elsewhere in driver and is not included in public API.